tests/benchmarks/benchmark.py

```diff
@@ -147,7 +147,7 @@ def test_HELIOTRON_run(tmpdir_factory, benchmark):
 @pytest.mark.benchmark(min_rounds=1, max_time=300, disable_gc=True, warmup=False)
 def test_HELIOTRON_vacuum_run(tmpdir_factory, benchmark):
     """Benchmark the HELIOTRON vacuum (fixed current) example."""
-    input_path = ".//tests//benchmarks//HELIOTRON_vacuum"
+    input_path = ".//tests//inputs//HELIOTRON_vacuum"
```

Comment:
Now that tests/benchmarks actually has these files, we should revert this change.